fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/RecurringDepositProductsApiResourceSwagger.java

```diff
@@ -343,88 +343,30 @@ private GetRecurringDepositProductsProductIdInterestCompoundingPeriodType() {}
             public String description;
         }
 
-        static final class GetRecurringDepositProductsProductIdAccountingMappings {
-
-            private GetRecurringDepositProductsProductIdAccountingMappings() {}
-
-            static final class GetRecurringDepositProductsProductIdSavingsReferenceAccount {
-
-                private GetRecurringDepositProductsProductIdSavingsReferenceAccount() {}
```

Comment:
Same here... what are the changes? there are lot of lines deleted...some added. Are you reordered them?

Reply:
I see, however if anyone was using the fineract-client to connect to fineract, now it would be a breaking change for them. Please revert this changes, and write a letter to dev mail list about the changes and the reasoning and let them be aware this is breaking change for anyone who is using fineract-client!